src/neo/Ledger/Blockchain.cs

```diff
@@ -244,9 +284,32 @@ private VerifyResult OnNewBlock(Block block)
                 // We can store the new block in block_cache and tell the new height to other nodes after Persist().
                 system.LocalNode.Tell(Message.Create(MessageCommand.Ping, PingPayload.Create(block.Index)));
```

Comment:
Do we still need to send `Ping` to others?

Reply:
I think Ping/Pong message can be removed

Reply:
Yes, we need to send Ping to others to tell latest height

Reply:
#1844